adal/src/main/java/com/microsoft/aad/adal/ADALError.java

```diff
@@ -554,7 +554,7 @@
     
     private String mDescription;
 
-    private ADALError(String message) {
+    ADALError(String message) {
```

Comment:
[http://docs.oracle.com/javase/tutorial/java/javaOO/enum.html](http://docs.oracle.com/javase/tutorial/java/javaOO/enum.html)
"Note: The constructor for an enum type must be package-private or private access. It automatically creates the constants that are defined at the beginning of the enum body. You cannot invoke an enum constructor yourself."
Don't know why AndroidStudio thinks private doesn't make sense. 

---
In reply to: [65658435](https://github.com/AzureAD/azure-activedirectory-library-for-android/pull/652#discussion_r65658435) [](ancestors = 65658435,65638135)